aztk/node_scripts/docker_main.sh

```diff
@@ -28,7 +28,8 @@ done
 echo "Starting setup using Docker"
 
 $(pyenv root)/versions/$AZTK_PYTHON_VERSION/bin/pip install -r $(dirname $0)/requirements.txt
-export PYTHONPATH=$PYTHONPATH:$DOCKER_WORKING_DIR
+
+echo 'export PYTHONPATH=$PYTHONPATH:$DOCKER_WORKING_DIR' >> ~/.bashrc
```

Comment:
don't you need to source bashrc now? or write the same line above

Reply:
yeah sourcing bash.rc is done in submit.py now: [here](https://github.com/Azure/aztk/pull/464/files#diff-05f414f140be1c8a088b89ddb5502a12R84)

Reply:
yeah but don't we also need it here for docker_main

Reply:
I don't think so, at least not until we do #465 but it won't hurt to do now.